.circleci/config.yml

```diff
@@ -36,10 +37,13 @@ jobs:
           sudo mkdir -p /go/src/istio.io/istio
           sudo chown -R circleci /go
       - checkout
-      - run: curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.7.4/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
-      - run: curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.24.1/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
+      - attach_workspace:
+          at:  /go
       - run:
+          name: "Start minikube"
           command: |
+            curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.7.4/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+            curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.22.3/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
```

Comment:
why do you need these?

Reply:
I didn't push the image yet ( bad karma to apply the changes before it is reviewed ), once the PR
is approved and image pushed I will remove this.